### ISSUE TITLE
JDK 24-ea build 22: Class com.sun.tools.javac.code.TypeTag does not have member field 'com.sun.tools.javac.code.TypeTag UNKNOWN

### ISSUE THREAD

**Issue by wendigo** [google/error-prone#4656]
```
[ERROR]      error-prone version: 2.35.1
Error:       BugPattern: (see stack trace)
Error:       Stack Trace:
Error:       java.lang.NoSuchFieldError: Class com.sun.tools.javac.code.TypeTag does not have member field 'com.sun.tools.javac.code.TypeTag UNKNOWN'
Error:    	at com.google.errorprone.util.ASTHelpers.<clinit>(ASTHelpers.java:1301)
Error:    	at com.google.errorprone.ErrorProneAnalyzer.shouldExcludeSourceFile(ErrorProneAnalyzer.java:267)
Error:    	at com.google.errorprone.ErrorProneAnalyzer.finished(ErrorProneAnalyzer.java:216)
Error:    	at jdk.compiler/com.sun.tools.javac.api.MultiTaskListener.finished(MultiTaskListener.java:133)
Error:    	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1432)
Error:    	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1379)
Error:    	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:964)
Error:    	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.lambda$doCall$0(JavacTaskImpl.java:104)
Error:    	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.invocationHelper(JavacTaskImpl.java:152)
Error:    	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:100)
Error:    	at jdk.compiler/com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:94)
Error:    	at org.codehaus.plexus.compiler.javac.JavaxToolsCompiler.compileInProcess(JavaxToolsCompiler.java:126)
Error:    	at org.codehaus.plexus.compiler.javac.JavacCompiler.performCompile(JavacCompiler.java:214)
Error:    	at org.apache.maven.plugin.compiler.AbstractCompilerMojo.execute(AbstractCompilerMojo.java:1228)
Error:    	at org.apache.maven.plugin.compiler.CompilerMojo.execute(CompilerMojo.java:215)
Error:    	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:126)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2(MojoExecutor.java:328)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:316)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:174)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.access$000(MojoExecutor.java:75)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor$1.run(MojoExecutor.java:162)
Error:    	at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute(DefaultMojosExecutionStrategy.java:39)
Error:    	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:159)
Error:    	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:105)
Error:    	at io.takari.maven.builder.smart.SmartBuilderImpl.buildProject(SmartBuilderImpl.java:206)
Error:    	at io.takari.maven.builder.smart.SmartBuilderImpl$ProjectBuildTask.run(SmartBuilderImpl.java:71)
Error:    	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
Error:    	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
Error:    	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
Error:    	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
Error:    	at java.base/java.lang.Thread.run(Thread.java:1576)
```

**Comment by wendigo** [comment]
Is it fixed by https://github.com/google/error-prone/pull/4642 ?

**Comment by cushon** [comment]
> Is it fixed by [#4642](https://github.com/google/error-prone/pull/4642) ?

Yes, that'll be included in the next release, you could also verify the fix with a snapshot if you like: https://oss.sonatype.org/content/repositories/snapshots/com/google/errorprone/

**Comment by wendigo** [comment]
Great, thanks for confirmation @cushon
